docs/topics/extraction.md

```diff
@@ -0,0 +1,16 @@
+# Extraction of Salt core modules
+TODO - ref https://github.com/salt-extensions/extension_migration
+
+## Gotchas
+### Unit test imports
+
+Unit tests import the modules directly. After migration, these imports
+need to be adjusted, otherwise the tests will run against the modules found in Salt, but still pass (or fail once they are removed). Example:
```

Comment:
This can be automated using [salt-rewrite](https://github.com/saltstack/salt-rewrite):

```shell
pip install git+https://github.com/saltstack/salt-rewrite
SALTEXT_NAME=vault salt-rewrite -F fix_saltext .
```

Reply:
Awesome, this should be part of the above TODO then.